README.md

```diff
@@ -108,7 +108,10 @@ Short option|Long option|Description
 (Not available)|`--self-update`|Update files to the latest release and quit
 (Not available)|`--singleplayer`|**DEPRECATED** Start singleplayer game, useful for save editing, using/testing DXVK in singleplayer, etc.
 (Not available)|`--skip-update-proton`|Skip updating already-installed Proton when updating game with Proton enabled
+(Not available)|`--steamruntimedir`|Choose a different Steam Runtime directory for Proton 5.13 or newer
+(Not available)|`--use-steam-runtime`|Use Steam Runtime when using Proton 5.13 or newer Note: If you want to use Discord Rich Presence, you need to start Discord with `XDG_RUNTIME_DIR=/path/to/dir` beforehand and also specify `--xdg-runtime-dir` option
```

Comment:
Shouldn't we make that the default for 5.13+ and allow the user to turn it off, basically the other way around?

Reply:
The current behavior is:

5.13 or newer?|with --use-steam-runtime?|Use Steam Runtime
---|---|---
Yes|Yes|Yes
Yes|No|No
No|Yes|No
No|No|No

* Is this help text confusing?
* Should we remove the Proton version check?

Reply:
The initial thing to do this whole runtime thing was because more recent proton versions didn't worked on not so recent operating systems. Hence we're trying to use the steam runtime to get a predictable environment which will (hopefully) always work.

IMHO we then should also make this the default so we can make use of this predictable environment for issue solving (hopefully we can replicate things easier) and user friendliness (all systems work ootb).

5.13 or newer?|with --without-steam-runtime?|Use Steam Runtime
---|---|---
Yes|Yes|No
Yes|No|Yes
No|Yes|No
No|No|No